content/docs/command-reference/plots/modify.md

```diff
@@ -0,0 +1,132 @@
+# plots modify
+
+Modify display properties of [plot metrics](/doc/command-reference/plots) files.
+
+## Synopsis
+
+```usage
+usage: dvc plots modify [-h] [-q | -v] [-t <path>] [-x X] [-y Y]
+                        [--no-csv-header] [--title <text>] [--x-label <text>]
+                        [--y-label <text>] [--unset [<prop> [<prop> ...]]]
+                        target
+
+positional arguments:
+  target                Plot file to set props to.
+```
+
+## Description
+
+It might be not convenient for users or automation systems to specify all the
+_display properties_ (such as `y-label`, `template`, `title`, etc.) each time
+plots are generated with `dvc plot show` or `dvc plot diff`. This command sets
+(or unsets) default display props for specific plot files.
+
+The available display properties are expressed as the [options](#options) of
+this command (prefixed with `--`).
+
+> Display props are based on the
+> [Vega specification](https://vega.github.io/vega/docs/specification/)
+
+The `target` metrics file should be an <abbd>output</abbr> of one of the
+[DVC pipeline](/doc/command-reference/pipeline) stages (see the `--plots` option
+of `dvc run`), listed in a
+[`dvc.yaml`](/doc/user-guide/dvc-files-and-directories) file. `dvc plots modify`
+adds the options to `dvc.yaml`.
+
+## Options
+
+- `-t <path>, --template <path>` - set a default
+  [plot template](/doc/command-reference/plots#plot-templates).
+
+- `-x <field>` - set a default field name from which the X axis data comes from.
+
+- `-y <field>` - set a default field name from which the Y axis data comes from.
+
+- `--x-label <text>` - set a default title for the X axis.
+
+- `--y-label <text>` - set a default title for the Y axis.
+
+- `--title <text>` - set a default plot title.
+
+- `--unset [<prop> [<prop> ...]]` - unset one or more display properties. Use
+  the prop name(s) without `--` in the argument sent to this option.
+
+- `--no-csv-header` - lets DVC know that CSV or TSV `targets` do not have a
```

Comment:
@dmpetrov @efiop @pared should it be just `--no-header`?

Reply:
how does behavior changes in this case - e.g. how do we determine default axis names? how do we convert such CSV/TSV to JSON internally? cc @pared

Reply:
or `--no-table-header` since TSV is also supported. This way it remains obvious that it only applies to tabular metric files. But I could also support `--no-header` — cleaner.

> how do we determine default axis names?

I'm not sure and couldn't check because of iterative/dvc/issues/3968 but I think the label for Y is just a number (the last one e.g. `4`) and for X is the same as usual: auto-generated `index` field.

> how do we convert such CSV/TSV to JSON internally?

I'm also assuming the field names are just the index numbers here: 0, 1, 2, etc. but would be great to confirm so we can add more info to this option description.

Reply:
I meant "column numbers"

Reply:
@jorgeorpinel I like the `--no-table-header`

@shcheklein internally, this switch is a boolean that tells `csv.DictReader` whether to treat first line as header or not. If not, columns are named by their index.

Reply:
Extracted option name discussion to iterative/dvc/issues/3996

I added some info. on the column names in 9e26518e.